apps/test/netsim/NetSimVizWire.js

```diff
@@ -26,33 +28,33 @@ describe("NetSimVizWire", function () {
       vizWire = new NetSimVizWire(localVizNode, remoteVizNode);
     });
 
-    it ("is a VizElement", function () {
+    it("is a VizElement", function () {
       assert(vizWire instanceof NetSimVizElement);
     });
 
-    it ("has default properties", function () {
-      assertEqual(0, vizWire.textPosX_);
-      assertEqual(0, vizWire.textPosY_);
-      assertEqual([], vizWire.encodings_);
+    it("has default properties", function () {
+      assert.equal(0, vizWire.textPosX_);
+      assert.equal(0, vizWire.textPosY_);
+      assert.deepEqual([], vizWire.encodings_);
       assert(localVizNode === vizWire.localVizNode);
```

Comment:
Oh!  I see.  `strictEqual` is more consistent with the original test, as we're checking object identity.